launchflow/workflows/tf/resources/gcp_firebase_hosting_site/variables.tf

```diff
@@ -0,0 +1,32 @@
+#
+# LaunchFlow global tofu variables
+#
+variable "gcp_project_id" {
+  type = string
+}
+
+variable "gcp_region" {
+  type = string
+}
+
+variable "resource_id" {
+  type = string
+}
+
+variable "environment_service_account_email" {
+  type = string
+}
+
+variable "artifact_bucket" {
+  type = string
+}
+
+
+variable "firebase_project_id" {
```

Comment:
nit: match the other styles with a comment seperating global from non global variables

Reply:
done